ssl/tls13_enc.c

```diff
@@ -389,6 +389,12 @@ static int derive_secret_key_and_iv(SSL_CONNECTION *s, const EVP_MD *md,
 
         if (mode == EVP_CIPH_GCM_MODE) {
             *taglen = EVP_GCM_TLS_TAG_LEN;
+        } else if (strcmp(EVP_CIPHER_get0_name(ciph), SN_enull_hmac_sha256)
+                   == 0) {
```

Comment:
@mattcaswell Is the tls_tag_len  something that could be handled as a gettable parameter (would it be cached in the cipher object via evp_cipher_cache_constants()) ?